src/navigation/Routes.android.js

```diff
@@ -1,6 +1,7 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import React, { useContext, useMemo } from 'react';
+import { PROFILES_ENABLED } from 'react-native-dotenv';
```

Comment:
just adding a note: would've originally expected this flag to live in `/config/experimental` - not sure if there have been discussions about dotenv being preferable

Reply:
👌 will add it there